atcodertools/common/language.py

```diff
@@ -52,7 +52,7 @@ def from_name(cls, name: str):
     name="cpp",
     display_name="C++",
     extension="cpp",
-    submission_lang_pattern=re.compile(".*C\\+\\+14 \\(GCC.*"),
+    submission_lang_pattern=re.compile(".*C\\+\\+ \\(GCC.*"),
```

Comment:
ここは既に他マージリクエストで対応したので衝突避けのために変更なしにします。
```suggestion
    submission_lang_pattern=re.compile(".*C\\+\\+14 \\(GCC.*"),
```